ChangeLog.d/issue4084.txt

```diff
@@ -0,0 +1,4 @@
+Removals
+    * Remove all support for MD2, MD4, RC4, Blowfish and XTEA. This removes the
+      corresponding modules and all their APIs and related configuration
```

Comment:
Please update the comment in `library/ssl_ciphersuites.c` that still mentions RC4 and 3DES.